packages/webcomponents/src/components/checkout/bank-account-form/bank-account-form.tsx

```diff
@@ -8,15 +8,17 @@ import { configState, waitForConfig } from "../../config-provider/config-state";
 export class BankAccountForm {
   @State() isReady: boolean = false;
   @State() iframeOrigin: string;
+  @State() tabId: string;
 
   private accountNumberIframeElement!: HTMLIframeInputElement;
   private routingNumberIframeElement!: HTMLIframeInputElement;
 
   async componentWillLoad() {
     await waitForConfig();
     this.iframeOrigin = configState.iframeOrigin;
+    this.tabId = crypto.randomUUID?.() || `${Date.now()}-${Math.random().toString(36).slice(2)}`;
```

Comment:
Is the fallback in case of older browsers that do not have `crypto.randomUUID`? If so, should we create a util function to share the fallback between card and bank account forms? It probably doesn't matter as long as it's unique, but would still keep from repeating this string template